packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -152,8 +152,48 @@ export class Blockchain {
   async getRawMempool (verbose: boolean): Promise<string[] | MempoolTx> {
     return await this.client.call('getrawmempool', [verbose], 'bignumber')
   }
+
+/** 
+ *  Get block statistics for a given window.
+ * @param {number} hashOrHeight  The block hash or height of the target block. 
+ * @param {Array<keyof BlockStats>} stats Default = all values. See BlockStats Interface. 
+ *@return {Promise<BlockStats>}  
+**/
+
+async getBlockStats(hashOrHeight: number | string, stats: Array<keyof BlockStats> = 
+    ["avgfee", "avgfeerate",
+     "avgtxsize", "blockhash",
+    "feerate_percentiles", "height",
+    "ins", "maxfee", "maxfeerate",
+    "maxtxsize", "medianfee",
+    "mediantime", "mediantxsize",
+    "minfee", "minfeerate",
+    "mintxsize", "outs", 
+    "subsidy", "swtotal_weight", 
+    "swtotal_size", "swtxs", "time", 
+    "total_out", "total_size", 
+    "total_weight", "totalfee", 
+    "txs", "utxo_increase", 
+    "utxo_size_inc"]
```

Comment:
use enum

Reply:
Figured out we don't need that anymore. Removed it.